firestore.rules

```diff
@@ -53,6 +53,11 @@ service cloud.firestore {
       allow update: if isAdmin() || (securityRelatedFieldsDidNotChange() && isOwnerOrClinicianOf(resource.data.user.organization));
     }
 
+    match /medicationClasses/{documents=**} {
```

Comment:
Can you make this a simple id, rather than applying those rules for all nested collections as well? Otherwise looks great, thank you!

Reply:
[Prevent applying rules for nested collections](https://github.com/StanfordBDHG/ENGAGE-HF-Firebase/pull/47/commits/cfafdb6b4e16d768d1fecd29411d4f00da2d06a1)